index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<title>Lily Boughton</title>
+		<link rel="stylesheet" href="normalize.css" type="text/css">
+		<link rel="stylesheet" href="styles.css" type="text/css">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
```

Comment:
move this above your styles.css (styles.css custom stuff should always come last)